sdk/python/tests/test_bigquery_ingestion.py

```diff
@@ -0,0 +1,149 @@
+import random
+import time
+from datetime import datetime, timedelta
+
+import pandas as pd
+import pytest
+from google.cloud import bigquery
+
+from feast.data_source import BigQuerySource
+from feast.feature import Feature
+from feast.feature_store import FeatureStore
+from feast.feature_view import FeatureView
+from feast.protos.feast.types.EntityKey_pb2 import EntityKey as EntityKeyProto
+from feast.protos.feast.types.Value_pb2 import Value as ValueProto
+from feast.repo_config import LocalOnlineStoreConfig, OnlineStoreConfig, RepoConfig
+from feast.value_type import ValueType
+
+
+@pytest.mark.integration
+class TestMaterializeFromBigQueryToDatastore:
+    def setup_method(self):
+        self.client = bigquery.Client()
+        self.gcp_project = self.client.project
+        self.bigquery_dataset = "test_ingestion"
+        dataset = bigquery.Dataset(f"{self.gcp_project}.{self.bigquery_dataset}")
+        self.client.create_dataset(dataset, exists_ok=True)
+        dataset.default_table_expiration_ms = (
+            1000 * 60 * 60 * 24 * 14
+        )  # 2 weeks in milliseconds
+        self.client.update_dataset(dataset, ["default_table_expiration_ms"])
+
+    def test_bigquery_table_to_datastore_correctness(self):
+        # create dataset
+        ts = pd.Timestamp.now(tz="UTC").round("ms")
+        checked_value = (
+            random.random()
```

Comment:
Can you please make sure we have a fixed seed? I really want to avoid flaky tests.

Reply:
this is a random value so that if nothing is written to the online store, we don't get a false positive through reading the result of the previous test; a fixed seed would get rid of that. I don't expect this to be flaky

Reply:
I know we arent planning to have flakyness, but sometimes tests evolve and non-determinism can creep in over time. Like if we start testing more types and there is conversion/casting. 

> we don't get a false positive through reading the result of the previous test

Can we isolate the tests to avoid this problem without using random values? Should we use different projects?